blocklylib-core/src/main/java/com/google/blockly/android/codegen/LanguageDefinition.java

```diff
@@ -11,7 +11,31 @@
      * Standard definition for the JavaScript language generator.
      */
     public final static LanguageDefinition JAVASCRIPT_LANGUAGE_DEFINITION
-            = new LanguageDefinition("javascript_compressed.js", "Blockly.JavaScript");
+            = new LanguageDefinition("generators/javascript_compressed.js", "Blockly.JavaScript");
+
+    /**
+     * Standard definition for the Lua language generator.
+     */
```

Comment:
Following up with the comment on the `build.gradle`, include a not in this and following doc comments that these non-JavaScript files are not included in core, and need to be added to your project manually. ("See demo build.gradle for an example.")